client/snapshot.go

```diff
@@ -176,3 +178,25 @@ func (client *Client) snapshotAction(action *snapshotAction) (changeID string, e
 
 	return client.doAsync("POST", "/v2/snapshots", nil, headers, bytes.NewBuffer(data))
 }
+
+// XXX: or should we just pass a filename in here ? this is how it was
+// done in 6ae9f7554fd5491468b1e148b45e483a084ea7b5 but it feels nicer to
```

Comment:
6ae9f7554fd5491468b1e148b45e483a084ea7b5 doesn't appear to be a valid commit id, can you double-check? In any case I agree, setID here makes sense.

Reply:
Hmm, interesting, apparently github resolved this commit id with a remark: "This commit does not belong to any branch on this repository."
I get an error locally:
```
$ git show 6ae9f7554fd5491468b1e148b45e483a084ea7b5
fatal: bad object 6ae9f7554fd5491468b1e148b45e483a084ea7b5
```

Reply:
it is a commit in @mvo5's branch: https://github.com/mvo5/snappy/commit/6ae9f7554fd5491468b1e148b45e483a084ea7b5

Reply:
Ah, I see, thanks. I think this comment should be removed when this PR lands.

Reply:
This is removed now